enclave_manager/avalon_enclave_manager/singleton/singleton_config.toml

```diff
@@ -0,0 +1,103 @@
+remote_url = "http://localhost:9090"
+
+# --------------------------------------------------
+# Logging -- configuration of service logging
+# --------------------------------------------------
+[Logging]
+LogLevel = "INFO"
+LogFile  = "__screen__"
+
+# ------------------------------------------------------------------
+# Work load execution-settings for workload execution(synchronous/asynchronous)
+# ------------------------------------------------------------------
+[WorkloadExecution]
+# Set this flag to 1 for sync workorder execution.Else set it to 0
+sync_workload_execution = 1
+
+# -------------------------------------------------------------------
+# EnclaveManager -- configuration of Intel SGX Enclave EnclaveManager
+# -------------------------------------------------------------------
+[EnclaveManager]
+# configurations for asynchronous work load execution
+sleep_interval = "10"
+# configurations for synchronous Work load execution
+# zmq_port is the port used for zmq socket communication
+# Port for socket interaction with listener
+zmq_port = '5555'
+
+# -------------------------------------------------------
+# EnclaveModule -- configuration of the Intel SGX Enclave
+# -------------------------------------------------------
+[EnclaveModule]
+# Service Provider ID (SPID) is a 32-digit hex string tied to the
+# enclave implementation. Replace dummy SPID with value obtained after
+# subscription to run TCS in Intel SGX HW mode.
+spid = "DEADBEEF00000000DEADBEEF00000000"
+
+num_of_enclaves = "1"
+
+# ias_url is the URL of the Intel Attestation Service (IAS) server.
+ias_url = "https://api.trustedservices.intel.com/sgx/dev"
+
+# Proxy for https. Leave commented out for direct Internet connections or
+# uncomment and change to your corporate proxy.
+#https_proxy = "http://your-proxy:your-port/"
+
+# IAS API key is a 32-digit hex string subscription key used for authentication
+# of requests submitted to the IAS server. Obtain the key by subscribing in
+# the portal https://api.portal.trustedservices.intel.com/
+ias_api_key = "112233445566778899aabbccddeeff00"
+
+# TEE enclave library to use.
+enclave_library = "libtcf-enclave.signed.so"
+enclave_library_path = "tc/sgx/trusted_worker_manager/enclave/build/lib/"
+
+# -------------------------------------------------------------
+# TCS Worker configuration details
+# -------------------------------------------------------------
+
+[WorkerConfig]
+ProofDataType = "TEE-SGX-IAS"
+
+# The Following URIs are used in direct model to submit work orders in
+# synchronous, asynchronous, pull and notification modes.
+WorkOrderSyncUri = ""
+WorkOrderAsyncUri = ""
+WorkOrderPullUri = ""
+WorkOrderNotifyUri = ""
+# Replace ApplicationTypeId and OrganizationId with real hexadecimal values
+ApplicationTypeId = "11aa22bb33cc44dd"
+OrganizationId = "aabbcc1234ddeeff"
+
+# URI used by worker to process work order receipts
+ReceiptInvocationUri = "http://localhost:8080"
```

Comment:
We don't support ReceiptInvocationUri now. If we are not using it please remove or keep it empty